promql/functions.go

```diff
@@ -1383,54 +1366,25 @@ func funcHistogramStdDev(vals []parser.Value, _ parser.Expressions, enh *EvalNod
 		}
 		enh.Out = append(enh.Out, Sample{
 			Metric:   sample.Metric,
-			F:        math.Sqrt(variance),
+			F:        varianceToResult(variance),
```

Comment:
Continuation of above suggestion:
```suggestion
			F:        variance,
```